src/appkit.cs

```diff
@@ -12332,21 +12678,27 @@ public interface NSSlider : NSAccessibilitySlider {
 		[Export ("altIncrementValue")]
 		double AltIncrementValue { get; set; }
 
+		[Deprecated (PlatformName.MacOSX, 10, 9)]
```

Comment:
/* These methods have never done anything, and are formally deprecated as of 10.9*/